google-http-client-xml/src/test/resources/heise-atom.xml

```diff
@@ -0,0 +1 @@
.cloudimg.io/width/264/q80.png-lossy-80.webp-lossy-80.foil1/_www-heise-de_/imgs/18/2/5/1/5/0/4/9/Low_Tech_Setup-ea7abd316395041b.png" alt="Low Tech Magazine Server mit Olimex Olinuxino" /></a><p>Energieeffizienz ist bei Webdesign bisher kein Thema. Das Low-Tech Magazine will das ändern und macht die Uptime der eigenen Website von der Sonne abhängig.</p></div></content></entry><entry><title type="text">Spiel mit der Sehnsucht: Online-Dating-Betrüger zu Haftstrafen verurteilt</title><id>https://heise.de/-4189378</id><updated>2018-10-11T19:09:00+02:00</updated><published>2018-10-11T19:09:00+02:00</published><link rel="alternate" type="text/html" href="https://www.heise.de/newsticker/meldung/Spiel-mit-der-Sehnsucht-Online-Dating-Betrueger-zu-Haftstrafen-verurteilt-4189378.html?wt_mc=rss.ho.beitrag.atom" /><summary type="html">In München werden drei Männer für Betrug mit erfundenen Identitäten beim Online-Dating verurteilt. Die Betroffenen fragen sich: Wie konnte mir das passieren?</summary><content type="xhtml"><div xmlns="http://www.w3.org/1999/xhtml"><a href="https://www.heise.de/newsticker/meldung/Spiel-mit-der-Sehnsucht-Online-Dating-Betrueger-zu-Haftstrafen-verurteilt-4189378.html?wt_mc=rss.ho.beitrag.atom" title="Spiel mit der Sehnsucht: Online-Dating-Betrüger zu Haftstrafen verurteilt"><img src="https://heise.cloudimg.io/width/264/q80.png-lossy-80.webp-lossy-80.foil1/_www-heise-de_/imgs/18/2/5/1/8/8/8/1/shutterstock_552542431-ab9661decfec96b8.jpeg" alt="Online-Dating" /></a><p>In München werden drei Männer für Betrug mit erfundenen Identitäten beim Online-Dating verurteilt. Die Betroffenen fragen sich: Wie konnte mir das passieren?</p></div></content></entry><entry><title type="text">Amazon: KI zur Bewerbungsprüfung benachteiligte Frauen</title><id>https://heise.de/-4189356</id><updated>2018-10-11T18:32:00+02:00</updated><published>2018-10-11T18:32:00+02:00</published><link rel="alternate" type="text/html" href="https://www.heise.de/newsticker/meldung/Amazon-KI-zur-Bewerbungspruefung-benachteiligte-Frauen-4189356.html?wt_mc=rss.ho.beitrag.atom" /><summary type="html">Eigentlich wollte Amazon eine Software entwickeln, die unter Bewerbern automatisch die besten findet. Der Algorithmus hatte aber unerwünschte Nebenwirkungen.</summary><content type="xhtml"><div xmlns="http://www.w3.org/1999/xhtml"><a href="https://www.heise.de/newsticker/meldung/Amazon-KI-zur-Bewerbungspruefung-benachteiligte-Frauen-4189356.html?wt_mc=rss.ho.beitrag.atom" title="Amazon: KI zur Bewerbungsprüfung benachteiligte Frauen"><img src="https://heise.cloudimg.io/width/264/q80.png-lossy-80.webp-lossy-80.foil1/_www-heise-de_/imgs/18/2/5/1/8/8/6/1/shutterstock_763283053-09be1eb5707c5ea5.jpeg" alt="Amazon: KI zur Bewerbungsprüfung benachteiligte Frauen" /></a><p>Eigentlich wollte Amazon eine Software entwickeln, die unter Bewerbern automatisch die besten findet. Der Algorithmus hatte aber unerwünschte Nebenwirkungen.</p></div></content></entry><entry><title type="text">Schlappe
```

Comment:
There may be copyright issues that prevent us from checking this in.

Reply:
Replaced with custom feed and added more validations